docs/usage_guide.rst

```diff
@@ -731,6 +731,58 @@ rotki automatically resolves ens name for each of your ethereum accounts. If the
    :alt: Example of resolving an ens name
    :align: center
 
+
+Ethereum Transactions
+=====================
+
+rotki is capable of getting and decoding (understand what is happening) your ethereum transactions. When you visit the ``Ethereum Transactions`` section the process to obtain all the information will start. You will be able to check
```

Comment:
Dude please apply the exact diff I am asking! 
https://github.com/rotki/rotki/pull/4374#discussion_r883872750

You fixed only 1 typo out of the 3

Reply:
Maybe I'm reading something bad but it has the 3 things you corrected committed